### PR TITLE
Add integration test scenarios for orchestrator-helm-operator

### PR DESCRIPTION
**Pull request by jordigilh**
Adds the integration tests for enterprise contract validation as described in the onboarding documentation.

### CODE REVIEW

File: cluster/stone-prd-rh01/tenants/orchestrator-releng-tenant/operator/integration_test_scenario.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+apiVersion: appstudio.redhat.com/v1beta1
+kind: IntegrationTestScenario
+metadata:
+  name: orchestrator-helm-operator-enterprise-contract
+spec:
+  params:
+    - name: POLICY_CONFIGURATION
+      value: rhtap-releng-tenant/tmp-onboard-policy
```
Comment (scoheb) [inline cluster/stone-prd-rh01/tenants/orchestrator-releng-tenant/operator/integration_test_scenario.yaml:8]: this exists?

Comment (jordigilh) [reply cluster/stone-prd-rh01/tenants/orchestrator-releng-tenant/operator/integration_test_scenario.yaml:8]: I'm not sure, I was following the onboarding documentation as it contains an example with this value. I've also seen it in these [2 other manifests](https://github.com/search?q=repo%3Aredhat-appstudio%2Ftenants-config%20rhtap-releng-tenant%2Ftmp-onboard-policy&type=code) for the insight-runtimes-tenant.

Comment (scoheb) [reply cluster/stone-prd-rh01/tenants/orchestrator-releng-tenant/operator/integration_test_scenario.yaml:8]: I see it!

**Comment by scoheb** [comment]
this PR has file conflicts

**Comment by jordigilh** [comment]
PR not needed. Closing it.

**Pull request closed without merging**